packages/patternfly-4/.babelrc

```diff
@@ -1,11 +0,0 @@
-{
-  "presets": ["@babel/react"],
-  "ignore": ["**/__snapshots__", "**/*.d.ts", "**/*.test.*"],
-  "plugins": [
-    "@babel/plugin-transform-typescript",
-    "babel-plugin-typescript-to-proptypes",
-    "@babel/plugin-proposal-export-default-from",
-    "@babel/proposal-class-properties",
-    "@babel/proposal-object-rest-spread",
-  ]
-}
```

Comment:
Why couldn't you have just renamed this to `babel.config.js` and added your plugin here? That would apply it to all the pf4 builds. I worked hard to unify our babel config and get away from `BABEL_ENV=cjs` or whatnot so that Windows works without having to use yet another tool called cross-env.

Soon I'm planning on removing babel in pf4 in favor of `tsc`, which we're already using for typechecking, so I'm also not really a fan of adding this plugin to begin with. Could we just change all the instances of `import { SomeIcon } from '@patternfly/react-icons';` to `import { SomeIcon } from '@patternfly/react-icons/dist/js/SomeIcon` in all `patternfly-4/*.{tsx,js}` files? There's only about 50 or so. We could easily add a linting step that gets upset if we try to use ESM imports like `import { SomeIcon } from '@patternfly/react-icons';`.

Reply:
If you are planing the direct imports that would be perfect and we can close this. I would prefer that anyway so you don't have to rely on some plugin.

I did not want to do any changes to the component code.

I would like to point out that the same thing should be done with react-core components in extensions like react-table.